readiness-tool/background.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * @fileoverview background.js provides a persistant background for the extension
+ * which individual extension instances that run on different tabs use to communicate.
+ */
+
+var globals = {};
+globals.tabToUrl = {};
+
+/**
+ * Send a message to page to do parse and process the DOM.
+ * @param {Tab} tab - Tab that needs to be processed
+ */
+function handleTab(tab) {
+
+  /**
+   * Delay everything by 2seconds, so that
+   * we are sure that javascript is ready to go.
```

Comment:
What JS are we waiting for here?

Reply:
The extension essentially does a regex match against innerHTML. What I notice is that many elements are dynamically inserted by javascript. Executing too early and you miss these elements, hence miss detecting these technologies. 

The 2s was a compromise, and there's no scientific reason for settling on that number.